care-pathways-data/ui/src/main/resources/PhenoTips/CarePathwaysConclusions.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version='1.0' encoding='UTF-8'?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+-->
+
+<xwikidoc version="1.1">
+  <web>PhenoTips</web>
+  <name>CarePathwaysConclusions</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1512151219000</creationDate>
+  <parent>PhenoTips.PatientSheet</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1512151219000</date>
+  <contentUpdateDate>1512151219000</contentUpdateDate>
+  <version>1.1</version>
+  <title/>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>false</hidden>
+  <content>{{include reference="PhenoTips.PatientSheetMacros" /}}
+
+{{include reference="PhenoTips.FormMacros" /}}
+
+{{velocity output=false}}
+$xwiki.ssx.use('PhenoTips.CarePathways')
+$xwiki.ssx.use('PhenoTips.PatientSheetCode', {'colorTheme' : "$xwiki.getSpacePreference('colorTheme')"})##
+$xwiki.ssx.use('PhenoTips.Entities', {'colorTheme' : "$xwiki.getSpacePreference('colorTheme')"})##
+$xwiki.jsx.use('PhenoTips.Entities')##
+##
+##
+#set ($displayDocExtra = false)
+##
+##
+#set($obj = $doc.getObject('PhenoTips.CarePathwaysConclusions', true))
+#set($class = $obj.xWikiClass)
+#set($discard = $doc.use($obj))
+## Disable the Preview button
+#set($previewenabled = false)
+{{/velocity}}
+
+{{velocity}}
+(% class="" id="care-pathways-conclusions-container"%)(((##
```

Comment:
If the class is empty, just don't mention it at all.

Reply:
Fixed.

Reply:
You didn't fix in the other 2 files...